src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,8 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 1; i <= 3; i++) {
+            System.out.println("Ball №" + i + ": " + new Lottery().getRandomBall());
```

Comment:
In your implementation, you creating Lottery 3 times that is not good approach. 
It's better to create `Lottery lottery = new Lottery();`
and use `lottery.getRandomBall()` in the for loop.

Also, create constant for 3 number